atoti-query-analyser/src/Components/Graph/Graph.js

```diff
@@ -105,16 +104,24 @@ class Graph extends Component {
 
     const { nodes, links } = buildD3(query, selectedRetrievals || selection);
 
+    console.log(nodes, links);
+
     const d3Graph = d3
       .select(ReactDOM.findDOMNode(this))
       .attr("width", window.innerWidth)
       .attr("height", window.innerHeight - 56);
 
     const clusters = _(nodes).map(n => n.clusterId);
+    console.log(clusters);
     const minC = clusters.min();
     const maxC = clusters.max();
     const viewCenter = window.innerWidth / 2;
-    const clusterCenter = parseInt((maxC - minC + 1) / 2 + minC - 0.5, 10); // Floor value
+    const clusterCenter = (minC + maxC) / 2;
```

Comment:
The issue that we have seen was related to the line 122 in this file. In the intermediate version I had `... / Math.max(1, maxC - minC)` instead of `... / 2`. This is why clusters were closer to each other than they should have been. The `clusterCenter` variable is responsible only for centering the graphics, not for its scaling.

Reply:
In fact, these formulae are pretty much the same, except of flooring :)